crypto/rand/rand_unix.c

```diff
@@ -510,6 +510,33 @@ size_t rand_pool_acquire_entropy(RAND_POOL *pool)
     bytes_needed = rand_pool_bytes_needed(pool, 1 /*entropy_factor*/);
     {
         size_t i;
+#ifdef DEVRANDOM_WAIT
+        static int wait_done = 0;
+
+        /*
+         * Ideally we would always pull entropy out of /dev/random,
+         * but that would quickly start to block if many prcesses
+         * compete for entropy. On the other hand, /dev/urandom
+         * is not guaranteed to be properly seeded when used too
+         * early after boot. So we use select to wait for /dev/random
+         * to be in readable state but do not read anything out of
+         * /dev/random. This has to be done only once.
```

Comment:
I suggest you reword it to say that on some implementations reading from /dev/urandom is possible before it's initialized, and that we wait for /dev/random to be readable to make sure it is initialized.

Reply:
okay.